app/models/vm_or_template.rb

```diff
@@ -69,6 +69,7 @@ class VmOrTemplate < ApplicationRecord
   has_one                   :miq_provision, :dependent => :nullify, :as => :destination
   has_one                   :miq_provision_template, :through => "miq_provision", :source => "source", :source_type => "VmOrTemplate"
   has_one                   :miq_server, :foreign_key => :vm_id, :inverse_of => :vm
+  has_one                   :server_group
```

Comment:
I think `belongs_to` makes more sense than `has_one` since the other side of the association is a `has_many`